S/SCIP_PaPILO/build_tarballs.jl

```diff
@@ -53,12 +54,12 @@ products = [
 ]
 
 dependencies = [
-    Dependency(PackageSpec(name="bliss_jll", uuid="508c9074-7a14-5c94-9582-3d4bc1871065"), v"0.77.0"),
-    Dependency(PackageSpec(name="boost_jll", uuid="28df3c45-c428-5900-9ff8-a3135698ca75"); compat="=1.76.0"),
+    Dependency(PackageSpec(name="bliss_jll", uuid="508c9074-7a14-5c94-9582-3d4bc1871065"), compat="=0.77.0"),
```

Comment:
To be clear, the message I linked suggested to use `=` compat _only_ for boost, because they break ABI in every single version, but that's not the case for BLIS and most of the other packages below, as far as I know. But below you do have boost, for that one you do need `=` 🙂

Reply:
bliss has had breaking changes all over the place and we don't need the bleeding edge in there so 077 forced seems wise